mip/var.go

```diff
@@ -0,0 +1,61 @@
+package mip
+
+// Var represents the entities on which the solver has to make a decision
+// without violating constraints and optimize the objective.
+// Vars can be of a certain type, binary, continuous or intehger.
+//
+// Continuous vars can take a value of a continuous quantity
+// Integer vars are vars that must take an integer value
+// (0, 1, 2, ...)
+// Binary vars can take two values, zero or one.
+type Var interface {
+	// Index is a unique number assigned to the var. The index corresponds
+	// to the location in the slice returned by Model.Variables().
+	Index() int
+	// IsBinary returns true if the invoking variable is a binary variable.
+	// otherwise false
```

Comment:
```suggestion
	// IsBinary returns true if the invoking variable is a binary variable,
	// otherwise it returns false.
```
Completing the sentence.